src/Core/Settings.h

```diff
@@ -436,6 +436,7 @@ class IColumn;
     M(Bool, optimize_normalize_count_variants, true, "Rewrite aggregate functions that semantically equals to count() as count().", 0) \
     M(Bool, optimize_injective_functions_inside_uniq, true, "Delete injective functions of one argument inside uniq*() functions.", 0) \
     M(Bool, convert_query_to_cnf, false, "Convert SELECT query to CNF", 0) \
+    M(Bool, optimize_or_like_chain, false, "Optimize multiple OR LIKE into multiMatchAny", 0) \
```

Comment:
Need to enable this setting value to `true`, for single CI run to verify tests. Will change it back to `false` before merge.